### PR TITLE
Use Selenium and Github Workflows for E2E tests

### PR DESCRIPTION
**Pull request by mrahanjam**
# Use Selenium and Github Workflows for E2E tests

## Description
This PR creates the github action/workflow to run the integration tests with selenium, it also includes a dummy test.

## PR Type
- [ ] Bug Fix
- [X] Feature
- [ ] Build Fix
- [ ] Testing
- [ ] General Improvement
- [ ] Cherry Pick

## Links
Jira: [CDAP-19260](https://cdap.atlassian.net/browse/CDAP-19260)

## Test Plan
N/A

## Screenshots
N/A

### CODE REVIEW

File: selenium/integration/starter.spec.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright © 2020 Cask Data, Inc.
```
Comment (seanfeld) [inline selenium/integration/starter.spec.ts]: is this a moved file? 2022

**Review by seanfeld — APPROVED** [review]
looks fine but i'm curious why you updated the node version for the test runner? I think its probably fine since its test only but we can't update node leswhere unfortunately :(

**Comment by mrahanjam** [comment]
> looks fine but i'm curious why you updated the node version for the test runner? I think its probably fine since its test only but we can't update node leswhere unfortunately :(

I think the main reason is to start the sandbox in the test env. It complains with node versions smaller than 12. ~I borrowed that part from the old workflow file.~ I believe 12 has better compatibility with the selenium-webdriver as well.

File: selenium/integration/starter.spec.ts (hunk 1 above)
Comment (mrahanjam) [reply selenium/integration/starter.spec.ts]: will update to 2022

File: selenium/integration/starter.spec.ts [hunk 2]
```diff
@@ -0,0 +1,45 @@
+/*
```
Comment (mrahanjam) [inline selenium/integration/starter.spec.ts:1]: This is a dummy test to make sure the webdriver works in the test env. Will be removed once we have real tests.

File: .github/workflows/node.js.yml [hunk 3]
```diff
@@ -36,14 +41,30 @@ jobs:
...
 
-    - name: Run Unit Tests
-      run: yarn run jest
+    - name: Start CDAP-UI and run the tests
+      run: |
+        Xvfb :1 -screen 0 1024x768x24 & export DISPLAY=:1 # Set Display port
+        yarn run cdap-full-build-more-memory # Build UI
+        yarn start & > server.out
+        yarn run test:unit
+        yarn run test:integration
```
Comment (njbriggs) [inline .github/workflows/node.js.yml]: Nit: newline before EOF

Comment (njbriggs) [reply .github/workflows/node.js.yml]: And a couple other files - please double-check

**Review by njbriggs — APPROVED** [review]

**Pull request merged**